hw/ip/dcd/data/dcd.hjson

```diff
@@ -0,0 +1,253 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "dcd",
+  clock_primary: "clk_i",
+  other_clock_list: [
+    "clk_aon_i",
+  ],
+  reset_primary: "rst_ni",
+  other_reset_list:
+  [
+    "rst_sw_ni"
+  ],
+  bus_device: "tlul",
+  bus_host: "none",
+  inter_signal_list: [
+    { struct: "adc",
+      type: "req_rsp",
+      act: "req",
+      name: "adc"
+      package: "dcd_pkg"
+    }
```

Comment:
Looks like you still missed the `pwr_sts` signal defined here.

Reply:
I removed pwr_sts signals in the RTL per Wednesday discussion. FW will write into CSR to determine the sampling rate and wakeup(from sleep mode).

-Eric